SpatialGDK/Extras/schema/debug_component.schema

```diff
@@ -5,4 +5,8 @@ component DebugComponent {
      id = 9995;
      option<uint32> worker_id_delegation = 1;
      string actor_tags = 2;
-}
\ No newline at end of file
+}
+
+component DebugComponentTag {
```

Comment:
why not use the actor tag here?

Reply:
Tried that, the query will actually mess with other sub views, by registering actors for which not all the components are available.

Reply:
ah- different queries, so different tag- makes sense